doc/readme.md

```diff
@@ -0,0 +1,337 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+# http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+-->
+
+#Cordova Command-line-interface (CLI) Reference
+
+##Syntax
+    
+    cordova command [options]
+
+##Global Commands
+
+These commands are available at all times.    
+
+| Command  | Description
+|----------|--------------
+| create | Create a project
+| help <command> | Get help for a command
+
+##Project Commands
+
+These commands are supported in a valid Cordova project.
+
+| Command      | Description
+|--------------|--------------
+| info         | Generate project information
+| requirements | Checks and print out all the installation requirements for platforms specified
+| platform     | Manage project platforms
+| plugin       | Manage project plugins
+| prepare      | Copy files into platform(s) for building
+| compile      | Build platform(s)
+| clean        | Cleanup project from build artifacts
+| run          | Run project (including prepare && compile)
+| serve        | Run project with a local webserver (including prepare)
+
+##Common options
+
+These options appply to all cordova-cli commands.
+
+| Option               | Description
+|----------------------|------------------------
+| -d or --verbose      | Pipe out more verbose output to your shell. You can also subscribe to `log` and `warn` events if you are consuming `cordova-cli` as a node module by calling `cordova.on('log', function() {})` or `cordova.on('warn', function() {})`.
+| -v or --version      | Print out the version of your `cordova-cli` install.
+| --no-update-notifier | will disable updates check. Alternatively set `"optOut": true` in `~/.config/configstore/update-notifier-cordova.json` or set `NO_UPDATE_NOTIFIER` environment variable with any value (see details in [update-notifier docs](https://www.npmjs.com/package/update-notifier#user-settings)).
+|--nohooks             | suppress executing hooks (taking RegExp hook patterns as parameters)
+
+##Examples
```

Comment:
I think that the format farther down in the platform examples where you intersperse the commands with their descriptions is much more readable